lambda/utils.py

```diff
@@ -1,12 +1,53 @@
 import json
+import logging
 from os.path import exists
 
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
 CONFIG_FILE = "config.json"
 
 
 def load_config() -> dict:
     if not exists(CONFIG_FILE):
-        raise ValueError(f"Config file does not exist")
+        raise ValueError("Config file does not exist")
 
     with open(CONFIG_FILE) as f:
         return json.load(f)
+
+
+def clean_chat_response(response: dict) -> str:
+    """Cleans the chat response from ChatGPT.
+
+    Args:
+        response (str): The response object from ChatGPT.
+
+    Returns:
+        str: The cleaned response.
+    """
+    error_messages = {
```

Comment:
Wonder how we could eventually make this translatable - I'll create a ticket

Reply:
Great! Thank you